dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -29,6 +29,19 @@ class Census::StateCsOffering < ApplicationRecord
     SC
   ).freeze
 
+  # By default we treat the lack of state data for high schools as an
+  # indication that the school doesn't teach cs. We aren't as confident
+  # that the state data is conplete for the following states so we do
+  # not want to treat the lack of data as a no for those.
+  INFERRED_NO_BLACKLIST = %w(
```

Comment:
could we use `EXCLUDELIST` or some other term that doesn't have potentially race-based etymology?

Reply:
I don't think there is anything race based about the term blacklist. I did a quick search of ye olde internet to confirm:
https://en.wikipedia.org/wiki/Blacklisting
https://www.quora.com/Is-the-term-blacklist-racist
https://english.stackexchange.com/a/51111

Reply:
I went ahead and changed it anyway.